frontend/src/routes/dashboard/layout.tsx

```diff
@@ -18,7 +18,8 @@ export default component$(() => {
   return (
     <div class="drawer drawer-mobile h-[calc(100vh-64px)]">
       <input id="drawer" type="checkbox" class="drawer-toggle" />
-      <div class="drawer-content w-100vh m-5" style={{ zIndex: -5 }}>
+      <div class="drawer-content w-100vh m-5">
+        {/* <div class="drawer-content w-100vh m-5" style={{ zIndex: -5 }}> */}
```

Comment:
Please remove the commented code

Reply:
- done: ([39c67ec](https://github.com/origranot/reduced.to/pull/396/commits/39c67ecfeb5c348e0a1a802d574d5b8a01f6bf43))

Reply:
- removed ([baa81f1](https://github.com/origranot/reduced.to/pull/396/commits/baa81f1509763e8d20345ae8ea9d68d102ad98ab))